app/sliders/importer.py

```diff
@@ -1,13 +1,16 @@
 from abc import ABC, abstractmethod
-import os
 from collections import defaultdict
 import json
 from openpyxl import load_workbook
 import pandas as pd
-# import logging
-# logging.basicConfig(level=logging.INFO)
+import logging
+import sys
+sys.path.append('/app/sliders/config_importer')
```

Comment:
Small refactoring detail:

Import statements should always go first, in three blocks:

1) Native Python modules
2) Extra Python modules (those libraries that we pip install)
3) Our own modules like the config_importer

Everything else, like the `sys.path.append()` and `logging.basicConfig()` lines, should come after

Reply:
Nice. Mach ich